repose-aggregator/commons/commons-utilities/src/main/java/org/openrepose/commons/utils/logging/apache/format/stock/ResponseBytesHandler.java

```diff
@@ -19,17 +19,32 @@
  */
 package org.openrepose.commons.utils.logging.apache.format.stock;
 
+import org.openrepose.commons.utils.StringUtilities;
```

Comment:
I'm going to replace these both. I wasn't sure if we were moved towards or away from this class.